test/sql/checks.test.lua

```diff
@@ -210,7 +211,12 @@ _ = s2:create_index('pk')
 s2:format({{name='X', type='any'}, {name='Y', type='integer'}})
 test_run:cmd("push filter 'space_id: [0-9]+' to 'space_id: <ID>'")
 _ = s1:create_check_constraint('physics', 'X < Y')
-_ = s1:create_check_constraint('physics', 'X > Y')
+
+--Show only index and space from ER_TUPLE_FOUND message
```

Comment:
What's the reason for that? Too long tuples?

Reply:
This test runs 2 times(under vinyl and under memtx) and one of tuple fields depends on the run number. One of the ways to fix it - don't show these tuples.

Reply:
Hm, it actually means that after first run some garbage is left. That's not good at all.
As a workaround you can just check the code of an error (error.code) and match with corresponding member from box.error

Reply:
Good idea. I've changed showing prefix of message to showing error.code and checking index and space names.